src/test/java/HelloTest.java

```diff
@@ -0,0 +1,12 @@
+import org.junit.Test;
```

Comment:
To jest przykładowy test, taki który zawsze się zaliczy. Możecie odpalić, by sprawdzić, czy wszystko dobrze skonfigurowane.
Skrót -> Ctrl + Shift + F10